openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/SuggestionRepository.java

```diff
@@ -220,6 +238,70 @@ protected void acceptSuggestion(
     update(suggestion, user);
   }
 
+  protected void acceptSuggestionList(
+      List<Suggestion> suggestions, SecurityContext securityContext, Authorizer authorizer) {
+    String user = securityContext.getUserPrincipal().getName();
+
+    // Entity being updated
+    EntityInterface entity = null;
+    EntityRepository<?> repository = null;
+    String origJson = null;
+    SuggestionWorkflow suggestionWorkflow = null;
+    SuggestionType suggestionType = getTypeFromSuggestions(suggestions);
```

Comment:
this should be a single type, what if user has suggestions for description and tags. This will merge them both